VERSION

```diff
@@ -1 +1 @@
-1.17.1-dev
+1.17.1
```

Comment:
Are you planning to kick off v1.17.1 release with this PR?

Reply:
yes https://github.com/envoyproxy/envoy/releases/tag/v1.17.1